xml/admin_security.xml

```diff
@@ -783,6 +783,223 @@ roleRef:
    </example>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="sec.admin.security.pod_policies">
+  <title>Pod Security Policies</title>
+  <para>
+   This section provides an overview of policy settings for pod
+   security. By default, pod security policies are already enabled on
+   &productname;.
+  </para>
+  <para>
+   &productname; comes with 2 defined policies. The required role
+   definition is also included in the examples. Role bindings have
+   to be created manually. For details about roles and role bindings,
+   refer to <xref linkend= "sec.admin.security.role" />.
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term><xref linkend="ex.admin.security.pod_policies.definition.unprivileged"/></term>
+    <listitem>
+     <para>
+      This is the default policy. It is a compromise between security
+      and daily needs. This policy is bound to the role
+      <literal>suse:caasp:psp:unprivileged</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><xref linkend="ex.admin.security.pod_policies.definition.privileged"/></term>
+    <listitem>
+     <para>
+      This policy has few restrictions and should only be given to
+      highly trusted users. This policy is bound to the role
+      <literal>suse:caasp:psp:privileged</literal>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   To adapt the provided example policies to your needs, copy them
+   into a YAML file. Then apply the definition by executing
+   <command>kubectl apply -f
+   <replaceable>YAML_FILE</replaceable></command>.
+  </para>
+  <para>
+   Detailed information is available at <link xlink:href=
+  "https://kubernetes.io/docs/concepts/policy/pod-security-policy/"
+  />.
+  </para>
+  <example xml:id="ex.admin.security.pod_policies.definition.unprivileged">
+   <title>Unprivileged Pod Security Policy</title>
+   <para>
+    The unprivileged Pod Security Policy is intended to be a
+    reasonable compromise between the reality of &kube; workloads and
+    the role <literal>suse:caasp:psp:privileged</literal>. By
+    default, &productname; grants this policy to all users
+    and service accounts.
+   </para>
+<screen>apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: suse.caasp.psp.unprivileged
```

Comment:
To avoid people copying + tweaking + applying, without changing the name, we should change the names used here to something like:

    examplecompany.psp.example-custom-psp